js-api-doc/value/color.d.ts

```diff
@@ -12,117 +90,456 @@ export class SassColor extends Value {
   /**
    * Creates an RGB color.
    *
-   * **Only** `undefined` should be passed to indicate a missing `alpha`. If
-   * `null` is passed instead, it will be treated as a [missing component] in
-   * future versions of Dart Sass. See [breaking changes] for details.
+   * If `space` is missing, **only** `undefined` should be passed to indicate a
+   * missing `alpha`. If `null` is passed instead, it will be treated as a
+   * [missing component] in future versions of Dart Sass. See [breaking changes]
+   * for details.
+   *
+   * If `space` is defined and `null` is passed for `alpha`, it will be treated
+   * as a [missing component]. In most cases, this is equivalent to the color
```

Comment:
We should probably document that passing null for _any_ component makes it missing.